src/contentful/utils.ts

```diff
@@ -42,16 +61,22 @@ export const getContentfulRequest = (type, contentQuery) => {
   };
 };
 
-const hasValidationErrors = (payload) => payload?.errors;
+const hasValidationErrors = (payload?: Record<string, unknown>) => payload?.errors;
```

Comment:
```suggestion
type ErrorResponse = Record<string, unknown> & { errors: unknown };`
const hasValidationErrors = (payload?: ErrorResponse | unknown) => Boolean(payload?.errors);
```

not sure here but i always create separate types to show error states

Reply:
https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates

Reply:
Didn't really got this one